README.md

```diff
@@ -47,9 +47,16 @@ var pg = require('pg')
 
 app.use(session({
   store: new pgSession({
-    pg : pg,                                  // Use global pg-module
-    conString : process.env.FOO_DATABASE_URL, // Connect using something else than default DATABASE_URL env variable
-    tableName : 'user_sessions'               // Use another table-name than the default "session" one
+    // When using pg v6 or above
+    pgPool : new pg.Pool({                       // Connection pool
+      // Insert pool options here
+    }),
```

Comment:
The example should probably create the `pg.Pool` object outside of the session store initialization, to make it clear that one should probably reuse the very same connection pool as one is using for the rest of the application.

Also: I would prefer to name this parameter just pool, that it relates to the database connection is pretty obvious